### DISCUSSION TITLE
Support more checkbox statuses

### DISCUSSION THREAD

**Discussion by jotaen** [jotaen/xit#65] [File Format]
Apart from the currently supported statuses `[ ]` (open), `[x]` (checked), `[@]` (ongoing), and `[~]` (obsolete), there might be more useful ones.

I think the overall number of statuses should be limited, just so that they can still be memorized relatively easily. To me, there shouldn’t be more than 5 or 6 overall, which – next to the existing ones – would leave room for 1–2 more.

The purpose of this issue is to collect and maintain an overview of potential candidates.

| Purpose | Symbol Ideas | |
|---|---|---|
|“unclear”, “in question”|`[?]`|https://github.com/jotaen/xit/discussions/34|
|“paused”, “on hold”|`[.]`|https://github.com/jotaen/xit/discussions/42|

Note: the requirements for a symbol are:

- It should be somewhat self-explanatory
- It should be language-agnostic, so it should be a symbol, not e.g. the first letter of an English word
- If ever possible, it should be an ASCII character, to ensure that it’s easy to type on every keyboard
- There should only be one canonical symbol per status, so no “aliases”

**Comment by clach04** [comment]
Refs:

* https://github.com/jotaen/xit/discussions/42
* https://github.com/jotaen/xit/discussions/34

**Comment by jotaen** [comment]
@clach04 thanks for the refs, I’ve added them to the table above.
